Initial rework of images (#7495)
kyma-project/test-infra: images/e2e-dind-k3d/README.md

# E2E DinD K3d

This image contains common tools for all jobs/tasks that test Kyma modules in K3d.

> K3d runs in docker-in-docker mode. Image is alpine-based.

Reviewer: ```suggestion
K3d runs in docker-in-docker mode. The image is Alpine-based.
```

> Usage of this image is restricted only for running K3d/kind workloads for testing.

Reviewer: ```suggestion
The usage of this image is restricted only to running K3d/kind workloads for testing.
```

It contains the following binaries:

* go
* k3d
* kind
* helm
* build-base
* ca-certificates
* curl
* bash
* jq
* yq
* docker
* docker-compose
* btrfs-progs
* e2fsprogs
* e2fsprogs-extra
* iptables
* xfsprogs
* xz
* git
* fuse-overlayfs
* device-mapper
* openssh-client
* dumb-init

## Usage

> 1. Use `securityContext: privileged: true` for this image to be running correctly.

Reviewer: ```suggestion
1. Use `securityContext: privileged: true` for this image to run correctly.
```

> 2. Do not define `command` directive in ProwJob or Pod definition. If you have to do it, use `/init.sh` as a value.

Reviewer: ```suggestion
2. Do not define the `command` directive in ProwJob or Pod definition. If you have to do it, use `/init.sh` as a value.
```
